docs/releasing/README.md

```diff
@@ -0,0 +1,337 @@
+1. Once an automatically created pull request gets merged, it will be deployed, you can then monitor:
+    - For Dispatch:
+        - [Deployment progress](https://app.datadoghq.com/container-images?query=short_image:dispatch)
+        - [Logs](https://app.datadoghq.com/logs?query=subnet%3Adispatch%20%40logger%3A%2A&live=true)
+        - [Metrics](https://app.datadoghq.com/dashboard/jrv-mm2-vuc/dispatch-testnet-subnets?live=true)
+    - For Echo:
+        - [Deployment progress](https://app.datadoghq.com/container-images?query=short_image:echo)
+        - [Logs](https://app.datadoghq.com/logs?query=subnet:echo%20@logger:*&live=true)
+        - [Metrics](https://app.datadoghq.com/dashboard/jrv-mm2-vuc/echo-testnet-subnets?live=true)
+
+    Note some metrics might be not showing up until a test transaction is ran.
+1. Launch a test transaction:
+    1. If you have no wallet setup, create a new one using the [Core wallet](https://core.app/)
+    1. Go to the settings and enable **Testnet Mode**
+    1. You need DIS (Dispatch) and ECH (Echo) testnet tokens. If you don't have one or the other, send your C-chain AVAX address to one of the team members who can send you some DIS/ECH testnet tokens. The portfolio section of the core wallet should then show the DIS and ECH tokens available.
+    1. For both Dispatch and Echo, in the "Command center", select **Send**, enter your own C-Chain AVAX address in the **Send To** field, set the **Amount** to 0 and click on **Send**. Finally, select a maximum network fee, usually *Slow* works, and click on **Approve**.
+1. You should then see the transaction impact the logs and metrics, for example
+
+    ```log
+    Apr 03 10:35:00.000 i-0158b0eef8b774d39 subnets Commit new mining work
+    Apr 03 10:34:59.599 i-0158b0eef8b774d39 subnets Resetting chain preference
+    Apr 03 10:34:56.085 i-0aca0a4088f607b7e subnets Served eth_getBlockByNumber
+    Apr 03 10:34:55.619 i-0ccd28afbac6d9bfc subnets built block
+    Apr 03 10:34:55.611 i-0ccd28afbac6d9bfc subnets Commit new mining work
+    Apr 03 10:34:55.510 gke-subnets-testnet subnets Submitted transaction
+    ```
+
+### Release
+
+If a successful release candidate was created, you can now create a release.
+
+Following the previous example in the [Release candidate section](#release-candidate), we will create a release `v0.7.3` indicated by the `$VERSION` variable.
+
+1. Head to the last release candidate pull request, and **restore** the deleted branch at the bottom of the Github page.
+1. Create a new release through the [Github web interface](https://github.com/ava-labs/subnet-evm/releases/new)
+    1. In the "Choose a tag" box, enter `$VERSION` (`v0.7.3`)
+    1. In the "Target", pick the previously restored last release candidate branch `releases/${VERSION_RC}`, for example `releases/v0.7.3-rc.0`.
```

Comment:
we should instead target a branch created from master, so the tag shows in the master log history, and further commits merged to master won't impact the release.